### PR TITLE
Working chat, and made something for the translationKey

### PR DESCRIPTION
**Pull request by STMiki**
- [x] Make something that works
- [x] Redo events
- [x] Add mode translation key
  - [x] chat.type.*
  - [x] Some more
- [x] Redo chat
- [x] Uncomment and fix the message in all commands

### CODE REVIEW

**Review by Trompettesib — APPROVED** [review]
GG WP

File: cubic-server/Entity.cpp [hunk 1]
```diff
@@ -57,6 +58,10 @@ void Entity::setRotation(uint8_t yaw, uint8_t pitch) { this->setRotation({yaw, p
 
 std::shared_ptr<Dimension> Entity::getDimension() const { return _dim; }
 
+World *Entity::getWorld() const { return _dim->getWorld(); }
```
[the remark this answers is not in the document]
Comment (Trompettesib) [reply cubic-server/Entity.cpp:61]: GL Huntears

[the remark this answers is not in the document]
Comment (STMiki) [reply cubic-server/Entity.cpp:61]: I was inspired by the const function above for the dimension, but the world is not a `shared_ptr`.
Maybe in the near future ? #137 :wink:

[the remark this answers is not in the document]
Comment (huntears) [reply cubic-server/Entity.cpp:61]: ~~I will commit die~~

**Review by huntears — APPROVED** [review]

**Pull request merged**